examples/test_application/src/main.rs

```diff
@@ -0,0 +1,451 @@
+//! Example usage of [`tss_ecdsa`] crate.
+//!
+//! Each [`Participant`] is represented by a worker thread. This example uses
+//! standard library shared memory MPSC channels to communicate [`Message`]s
```

Comment:
suggestion: maybe link to the mpsc page here for those who don't know what this is? https://doc.rust-lang.org/std/sync/mpsc/